ouroboros-consensus-diffusion/ouroboros-consensus-diffusion.cabal

```diff
@@ -88,7 +88,7 @@ library
     , filepath
     , fs-api
     , hashable
-    , io-classes                   ^>=0.3
+    , io-classes                   ^>=0.3     || ^>=0.5
```

Comment:
```suggestion
    , io-classes                   ^>=0.3 && <0.6
```

Reply:
:thinking: or maybe you want to be more conservative in the consensus side?

Reply:
I think so :)